### PR TITLE
pom.xml missing references fix

### PR DESCRIPTION
**Pull request by Neves97**
Fixed missing maven references on Pet-Store sample project

### CODE REVIEW

File: samples/springboot3/pet-store/pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
         <dependency>
             <groupId>com.amazonaws.serverless</groupId>
             <artifactId>aws-serverless-java-container-springboot3</artifactId>
-            <version>[2.0-SNAPSHOT,)</version>
+            <version>2.0.0-M2</version>
```
Comment (deki) [inline samples/springboot3/pet-store/pom.xml:46]: Let's still utilize a version range. `[2.0.0-SNAPSHOT,),[2.0.0-M1,)` should do the trick (related issue https://issues.apache.org/jira/browse/MNG-3092).

File: samples/springboot3/pet-store/pom.xml [hunk 2]
```diff
@@ -55,7 +55,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-shade-plugin</artifactId>
-                        <version>3.5.1</version>
+                        <version>3.5.0</version>
```
Comment (deki) [inline samples/springboot3/pet-store/pom.xml:58]: No need to downgrade or were you facing any issues with the latest version?

**Review by deki — CHANGES_REQUESTED** [review]

File: samples/springboot3/pet-store/pom.xml (hunk 2 above)
Comment (Neves97) [reply samples/springboot3/pet-store/pom.xml:58]: >3.5.0  Was being marked as not found, so I downgraded to the suggested version.
`Plugin 'org.apache.maven.plugins:maven-shade-plugin:3.5.1' not found`

Comment (Neves97) [reply samples/springboot3/pet-store/pom.xml:58]: I was able to build just fine though, but the warning does not go away.

Comment (deki) [reply samples/springboot3/pet-store/pom.xml:58]: Maybe related to your local cache? It's available in central: https://repo1.maven.org/maven2/org/apache/maven/plugins/maven-shade-plugin/3.5.1/
Can you please remove the change from the PR? I will merge afterwards. Thanks.

**Pull request merged**